### PR TITLE
docs: retro fit alerts to include args annotations

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

Start retro fitting commands to include docs for args 

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/alerts/global_list.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func GlobalListBuilder() *cobra.Command {
 	opts.Template = listTemplate
 	cmd := &cobra.Command{
 		Use:     "list",
-		Short:   "List Ops Manager global alerts.",
+		Short:   "Retrieves all the global alerts for the specified Ops Manager project.",
```
Comment (andreaangiolillo) [inline internal/cli/alerts/global_list.go:64]:
```suggestion
		Short:   "Retrieves all the global alerts for the specified project.",
```

**Review by andreaangiolillo — CHANGES_REQUESTED** [review]
One small change. Otherwise LGTM

File: internal/cli/alerts/global_list.go (hunk 1 above)
Comment (gssbzn) [reply internal/cli/alerts/global_list.go:64]: this command is OM only ;)

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Review by fmenezes — APPROVED** [review]

File: internal/cli/alerts/acknowledge.go [hunk 2]
```diff
@@ -88,6 +88,10 @@ func AcknowledgeBuilder() *cobra.Command {
 				opts.InitOutput(cmd.OutOrStdout(), ackTemplate),
 			)
 		},
+		Annotations: map[string]string{
```
Comment (fmenezes) [inline internal/cli/alerts/acknowledge.go:91]: ❤️

**Review by melissamahoney-mongodb — APPROVED** [review]

**Pull request merged**